server/repository/migrations/postgres/2022-06-21-013225_create_log_table/up.sql

```diff
@@ -6,7 +6,14 @@ CREATE TYPE log_type AS ENUM (
     'INVOICE_STATUS_PICKED',
     'INVOICE_STATUS_SHIPPED',
     'INVOICE_STATUS_DELIVERED',
-    'INVOICE_STATUS_VERIFIED'
+    'INVOICE_STATUS_VERIFIED',
+    'STOCKTAKE_CREATED',
+    'STOCKTAKE_DELETED',
+    'STOCKTAKE_STATUS_FINALISED',
+    'REQUISITION_CREATED',
+    'REQUISITION_DELETED',
+    'REQUISITION_STATUS_SENT',
+    'REQUISITION_STATUS_FINALISED'
 );
 
 CREATE TABLE log (
```

Comment:
the type column is incorrect..
```
CREATE TABLE log (
    id TEXT NOT NULL PRIMARY KEY,
    type log_type, 
    user_id TEXT,
    store_id TEXT REFERENCES store(id),
    record_id TEXT,
    datetime TIMESTAMP NOT NULL
)
```

for sqlite and in the rest of the codebase it is `log_type` 
I actually prefer type, since we are in the log table, but expect the rename is due to reserved word issues?

Reply:
Type isn't reserved it appears 🤔 
Will rename 😄

Reply:
Will change to the same naming as the other tables 😄